modular_skyrat/modules/conveyor_sorter/code/conveyor_sorter.dm

```diff
@@ -145,3 +149,43 @@
 		"conveysorter",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+
+/obj/item/conveyor_sorter/improved
+	name = "improved conveyor sorter lister"
+	desc = "A tool that is used to not only create the conveyor sorters, but give lists to the conveyor sorters."
+	icon_state = "lister_improved"
+	max_sorters = 8
+	max_items = 10
+	conveyor_type = /obj/effect/decal/cleanable/conveyor_sorter/improved
+
+/obj/effect/decal/cleanable/conveyor_sorter/improved
```

Comment:
I've got just one question...
Since it's a subtype of `cleanable` does it mean you can literally just clean it away? That sounds kind of cursed.

Reply:
I am not entirely sure, its how the convayor sorter was before that would more be a question for @jjpark-kb

Reply:
@GoldenAlpharex do you want me to change it to not be cleanable or is it fine as is?